manifests/db.pp

```diff
@@ -67,6 +68,10 @@
         require     => Mysql_grant["${user}@${host}/${table}"],
         subscribe   => Mysql_database[$dbname],
         timeout     => $import_timeout,
+        onlyif      => $onlyif_empty ? {
+          true    => "test $(mysql ${dbname} -e 'show tables' |wc -l ) -eq 0",
+          default => "false",
```

Comment:
It's a false positive: I'm not really quoting a boolean value - it's a call to /bin/false.